util/appveyor/build.cmd

```diff
@@ -0,0 +1,21 @@
+@echo off
+:: To build extensions for 64 bit Python 3, we need to configure environment
+:: variables to use the MSVC 2010 C++ compilers from GRMSDKX_EN_DVD.iso of:
+:: MS Windows SDK for Windows 7 and .NET Framework 4
+::
+:: More details at:
+:: https://github.com/cython/cython/wiki/64BitCythonExtensionsOnWindows
```

Comment:
Nice, wasn't aware of those docs. I see now that Python 3.5 doesn't use that same SDK, switched via ``DISTUTILS_USE_SDK``. So looks good to me.